packages/captp/lib/captp.js

```diff
@@ -291,15 +320,20 @@ export function makeCapTP(ourId, rawSend, bootstrapObj = undefined) {
 
   // Return a dispatch function.
   const dispatch = obj => {
-    if (unplug !== false) {
+    try {
+      if (unplug !== false) {
+        return false;
+      }
+      const fn = handler[obj.type];
+      if (fn) {
+        fn(obj).catch(e => quietReject(e, false));
+        return true;
+      }
+      return false;
+    } catch (e) {
+      quietReject(e);
```

Comment:
If I'm deducing the purpose of `returnIt` correctly, shouldn't this one be `quietReject(e, false)`? I guess it doesn't matter too much, the rejected promise created by `returnIt` defaulting to `true` has a handler attached, so it won't provoke a Node.js warning, but it seems like we use `returnIt = false` in all the other cases where `quietReject` is called and its return value is ignored.

Reply:
Good catch!  Yes, you're right.